### PR TITLE
Expose Html.to_plain_text as Omd.to_string

### PR DESCRIPTION
**Pull request by cuihtlauac**
Expose function allowing to turn an inline element into a string,
keeping only the text.

This is fix for issue https://github.com/ocaml/omd/issues/299

### CODE REVIEW

File: src/omd.mli [hunk 1]
```diff
@@ -30,3 +30,6 @@ val of_channel : in_channel -> doc
 val of_string : string -> doc
 val to_html : ?auto_identifiers:bool -> doc -> string
 val to_sexp : doc -> string
+
+val to_string : 'attr inline -> string
```
Comment (shonfeder) [inline src/omd.mli:34]: Perhaps this should be
 
```suggestion
val to_plain_text : 'attr inline -> string
```

to avoid confusion?

**Review by shonfeder — COMMENTED** [review]
Thanks!

What do you think of my alternative naming suggestion?

**Comment by cuihtlauac** [comment]
It is much better indeed. Can you do it? I'm afk this week.

File: src/omd.ml [hunk 2]
```diff
@@ -32,3 +32,4 @@ let to_html ?auto_identifiers doc =
   Html.to_string (Html.of_doc ?auto_identifiers doc)
 
 let to_sexp ast = Format.asprintf "@[%a@]@." Sexp.print (Sexp.create ast)
+let to_string = Html.to_plain_text
```
Comment (shonfeder) [inline src/omd.ml:35]:
```suggestion
let to_plain_text = Html.to_plain_text
```

**Pull request closed without merging**